python/taichi/lang/snode.py

```diff
@@ -399,12 +436,13 @@ def get_addr(f, indices):
     Currently, this function can only be called inside a taichi kernel.
 
     Args:
-        f (Union[ti.field, ti.Vector.field, ti.Matrix.field]): Input taichi field for memory address query.
-        indices (Union[int, ti.Vector()]): The specified field indices of the query.
+        f (Union[:class:`~taichi.field`, :class:`~taichi.Vector.field`, \
```

Comment:
`taichi.field` is not a class - `taichi.Field` is. This should be corrected to get the hyperlink work.

Reply:
`taichi.Matrix.field` should become `taichi.MatrixField` accordingly.

Reply:
Corrected.

Reply:
`taichi.Vector.field` is also `taichi.MatrixField`.

Reply:
corrected

Reply:
As the two `taichi.MatrixField`s are the same, shall we not duplicate them?

Reply:
you are right, I'm sorry for this careless mistake.